src/main/resources/templates/pages/featureOverview.vm

```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
+#if($fromJenkins)
+	<base href="${jenkins_base}job/$build_project/$build_number/cucumber-html-reports/">
```

Comment:
As I said, that's fine to me. Keep going, resolve conflict and pull corrected PR ;)